iiwa_driver/src/iiwa.cpp

```diff
@@ -304,8 +305,16 @@ namespace iiwa_ros {
         _fri_message_data->resetCommandMessage();
 
         if (_robot_state.getClientCommandMode() == kuka::fri::TORQUE) {
+            _joint_position_command = _joint_position;
+            // Implements dithering to trigger the friction observer.
+            // If the physical robot is at the commanded positions, KUKA turns off friction
+            // compensation.
+            for (int i = 0; i < 7; i++)
```

Comment:
Also, I think we should make this `size_t` instead of `int`...

Reply:
Sure. I also fixed that in other points of the driver. I didn't touch `iiwa_tools` and `iiwa_control` though